.travis.yml

```diff
@@ -8,7 +10,9 @@ before_script:
 
 # Assume bundler is being used, therefore
 # the `install` step will run `bundle install` by default.
-script: ./ci/cibuild
+script:
+  - ./ci/cibuild
+  - docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci
```

Comment:
is the /app actually working in this case? Shouldn't it point to the projects directory?

Reply:
See User Docker documentation